tools/dockerise/__init__.py

```diff
@@ -243,23 +164,48 @@ def run(rebuild, clean, platform, **kwargs):
                     build_platform(platform)
                     # If we were building the selected platform we have to move our selected tag up
                     if selected_platform:
-                        client.images.get(tag).tag(repository, "selected")
+                        try:
+                            subprocess.Popen(
+                                ["docker", "image", "tag", tag, "{}:selected".format(repository)],
+                                stdout=subprocess.DEVNULL,
+                            ).communicate()
+                        except subprocess.CalledProcessError:
+                            cprint("docker image tag returned a non-zero exit code", "red", attrs=["bold"])
+                            exit(1)
 
                 # Find the volume for this platform
                 build_volume_name = "{}_{}_build".format(repository, platform)
                 try:
-                    build_volume = client.volumes.get(build_volume_name)
+                    subprocess.Popen(
+                        ["docker", "volume", "inspect", build_volume_name], stdout=subprocess.DEVNULL
+                    ).communicate()
+
+                    build_volume = build_volume_name
+                except subprocess.CalledProcessError:
+                    build_volume = None
+
+                # If we are cleaning, remove this volume so we can recreate it
+                if build_volume and clean:
+                    try:
+                        subprocess.Popen(
+                            ["docker", "volume", "rm", build_volume_name], stdout=subprocess.DEVNULL
+                        ).communicate()
+                    except subprocess.CalledProcessError:
+                        cprint("Docker volume rm returned a non-zero exit", "red", attrs=["bold"])
+                        exit(1)
 
-                    # If we are cleaning, remove this volume so we can recreate it
-                    if clean:
-                        build_volume.remove(force=True)
-                        build_volume = None
-                except docker.errors.NotFound:
                     build_volume = None
 
                 # If we don't have a volume, make one
                 if build_volume is None:
-                    build_volume = client.volumes.create(build_volume_name)
+                    try:
+                        subprocess.Popen(
+                            ["docker", "volume", "create", build_volume_name], stdout=subprocess.DEVNULL
+                        ).communicate()
```

Comment:
This also could probably just be an if statement with `subprocess.call` rather than a try-catch block